lutris/gui/config_dialogs.py

```diff
@@ -29,6 +29,18 @@ def do_insert_text(self, new_text, length, position):
             return position + length
         return position
 
+class NumberEntry(Gtk.Entry, Gtk.Editable):
```

Comment:
There should be an additional blank line before the class definition: https://www.python.org/dev/peps/pep-0008/#blank-lines